tests/foreman/ui/test_settings.py

```diff
@@ -209,53 +211,58 @@ def test_positive_selectors(session):
                 setting_cleanup(setting_attr, str(original_val.default))
 
 
-@stubbed()
 @tier3
-def test_positive_update_login_page_footer_text_with_long_string():
-    """Attempt to update parameter "Login_page_footer_text"
-        with long length string under General tab
+def test_positive_update_login_page_footer_text_with_long_string(session):
+    """Testing to update parameter "Login_page_footer_text with long length
+    string under General tab
 
     :id: b1a51594-43e6-49d8-918b-9bc306f3a1a2
 
-    :steps:
```

Comment:
Shouldn't we keep these test steps?

Reply:
I think it is not required here, because the steps are very straight forward.
If you think it should be then definitely I will update it.

Reply:
ok.

Reply:
fixed.